<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsLayoutImageExportOptionsDialog</class>
 <widget class="QDialog" name="QgsLayoutImageExportOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Export Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QgsCollapsibleGroupBoxBasic" name="groupBox">
     <property name="title">
      <string>Export Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" colspan="5">
       <widget class="QCheckBox" name="mGenerateWorldFile">
        <property name="toolTip">
         <string>If checked, a separate world file which georeferences exported images will be created</string>
        </property>
        <property name="text">
         <string>Generate world file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Page width</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QgsSpinBox" name="mWidthSpinBox">
        <property name="specialValueText">
         <string>Auto</string>
        </property>
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>99999999</number>
        </property>
        <property name="showClearButton" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Export resolution</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="5">
       <widget class="QCheckBox" name="mAntialiasingCheckBox">
        <property name="toolTip">
         <string>If unchecked, the generated images will not be antialiased</string>
        </property>
        <property name="text">
         <string>Enable antialiasing</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QgsSpinBox" name="mResolutionSpinBox">
        <property name="suffix">
         <string> dpi</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>3000</number>
        </property>
        <property name="showClearButton" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Page height</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QgsSpinBox" name="mHeightSpinBox">
        <property name="specialValueText">
         <string>Auto</string>
        </property>
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>99999999</number>
        </property>
        <property name="showClearButton" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mQualityLabel">
        <property name="text">
         <string>Quality</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSlider" name="mQualitySlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>90</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="mQualitySpinBox">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>90</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBoxBasic" name="mClipToContentGroupBox">
     <property name="title">
      <string>Crop to Content</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Left</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QgsSpinBox" name="mLeftMarginSpinBox">
            <property name="suffix">
             <string/>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Right</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QgsSpinBox" name="mRightMarginSpinBox">
            <property name="suffix">
             <string/>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Bottom</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Top margin (px)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QgsSpinBox" name="mTopMarginSpinBox">
          <property name="suffix">
           <string/>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QgsSpinBox" name="mBottomMarginSpinBox">
          <property name="suffix">
           <string/>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QDialogButtonBox" name="mHelpButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Help</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="mOpenAfterExportingCheckBox">
       <property name="text">
        <string>Open file after exporting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBoxBasic</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mResolutionSpinBox</tabstop>
  <tabstop>mWidthSpinBox</tabstop>
  <tabstop>mHeightSpinBox</tabstop>
  <tabstop>mQualitySlider</tabstop>
  <tabstop>mQualitySpinBox</tabstop>
  <tabstop>mAntialiasingCheckBox</tabstop>
  <tabstop>mGenerateWorldFile</tabstop>
  <tabstop>mClipToContentGroupBox</tabstop>
  <tabstop>mTopMarginSpinBox</tabstop>
  <tabstop>mLeftMarginSpinBox</tabstop>
  <tabstop>mRightMarginSpinBox</tabstop>
  <tabstop>mBottomMarginSpinBox</tabstop>
  <tabstop>mOpenAfterExportingCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsLayoutImageExportOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsLayoutImageExportOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
